tests/ui/zero_repeat_side_effects.fixed

```diff
@@ -40,9 +41,9 @@ fn main() {
     //~^ zero_repeat_side_effects
 
     // when singled out/not part of assignment/local
-    { f(); vec![] as std::vec::Vec<i32> };
+    f(); vec![] as std::vec::Vec<i32>;
```

Comment:
Nit: can't you put this on two lines and repeat the indentation? It would probably look cleaner.